alpine-nix-rails-nginx/etc/sv/nginx

```diff
@@ -3,4 +3,6 @@
 set -o errexit -o pipefail -o noglob -o noclobber -o nounset
 IFS=$'\n\t'
 
-exec nginx -c /etc/nginx/nginx.conf -g 'daemon off;'
+export PATH="/nix/var/nix/profiles/default/bin:/nix/var/nix/profiles/default/sbin:$PATH"
```

Comment:
Will this inherit from the docker environment, or the user-that-bundles-the-docker-image environment.